can_tools/scrapers/usafacts/data.py

```diff
@@ -30,41 +30,44 @@ def fetch(self) -> pd.DataFrame:
         return pd.read_csv(BASEURL + self.filename)
 
     def normalize(self, data: Any) -> pd.DataFrame:
+        # Make lowercase so they can't change capitalization on us
+        data.columns = [c.lower() for c in data.columns]
```

Comment:
USAFacts changed the capitalization of their columns -- This prevents us from being exposed to a future change in capitalization.